app/javascript/dashboard/dashboard.jsx

```diff
@@ -4,16 +4,23 @@ import displayClick from '../../assets/images/display_click.png';
 import displayTime from '../../assets/images/display_time.png';
 import displayComment from '../../assets/images/display_comment.png';
 
-
 export class Dashboard extends Component {
   constructor(props) {
     super(props);
     this.state = {};
   }
 
   render() {
-    /* eslint-disable */ const { visits, visit, comments, time, articles, avgArticles, words, avgWords } = this.props;
-console.log(avgArticles)
+    /* eslint-disable */ const {
+      visits,
+      visit,
+      comments,
+      time,
+      articles,
+      avgArticles,
+      words,
+      avgWords,
```

Comment:
Same for this line, I assume using camel case doesn't break anything?

Reply:
Yes, it was only the HTML data attributes where we were having the casing issues.